.github/CHANGELOG.md

```diff
@@ -41,13 +46,16 @@ dev2 = qml.device('lightning.qubit', wires=4, c_dtype=np.complex128) # for doubl
 ### Bug fixes
 
 * Fix the issue with using available `clang-format` version in format.
-[#(288)](https://github.com/PennyLaneAI/pennylane-lightning/pull/288)
+[(#288)](https://github.com/PennyLaneAI/pennylane-lightning/pull/288)
+
+* Fix a bug in the generator of `DoubleExcitationPlus`.
+[(#298)](https://github.com/PennyLaneAI/pennylane-lightning/pull/298)
 
 ### Contributors
 
 This release contains contributions from (in alphabetical order):
 
-Ali Asadi, Amintor Dusko, Chae-Yeun Park, Lee James O'Riordan, Mikhail Andrenkov
+Ali Asadi, Amintor Dusko, Chae-Yeun Park, Lee James O'Riordan, Mikhail Andrenkov, Shuli Shu
```

Comment:
I have never seen we sorted the names correctly (by the last name), but we may try this time :smile: 
```suggestion
Mikhail Andrenkov, Ali Asadi, Amintor Dusko, Lee James O'Riordan, Chae-Yeun Park, and Shuli Shu
```